docs/src/pages/discover-more/showcase/Showcase.js

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import Link from 'docs/src/modules/components/Link';
+import GithubIcon from '@material-ui/docs/svgIcons/GitHub';
+
+const styles = theme => ({
+  gridContainer: {
+    backgroundColor: theme.palette.background.default,
+    // Hide the demo container padding
+    margin: -24,
+    width: 'calc(100% + 48px)',
+    // Maintain alignment with the markdown text
+    [theme.breakpoints.down('xs')]: {
+      padding: 30,
+    },
+  },
+  card: {
+    marginTop: theme.spacing.unit * 2,
+    marginBottom: theme.spacing.unit * 6,
+  },
+  cardMedia: {
+    paddingTop: '75%', // 4:3
+  },
+});
+
+const appList = [
+  {
+    title: 'Venumob',
+    description:
+      'A platform for individuals and businesses to find and book the perfect venue for any event.',
+    image: '/static/images/showcase/venuemob.jpg',
+    link: 'https://venuemob.com.au/',
+  },
+  {
+    title: 'Local Insights',
+    description:
+      'A real estate data provider that aggregates and analyzes property records, ' +
+      'permits, and tax documents.',
+    image: '/static/images/showcase/localinsights.jpg',
+    link: 'https://localinsights.io',
+    source: 'https://github.com/local-insights',
```

Comment:
How is that a source?

Reply:
Perhaps it was originally but then made private?